src/routes/index.js

```diff
@@ -1,6 +1,8 @@
 const Joi = require('joi');
 
 const movieController = require('../controllers/movieController');
+const movieResponseObj = require('../models/movieResponse');
+const movieDtlResponseObj = require('../models/movieDtlResponse');
```

Comment:
movieDtl stand for movie detail, anyway I changed it to clearer variable name: movieDetailResponseObj